modules/installation-mirror-repository.adoc

```diff
@@ -55,8 +56,43 @@ the pull secret for your mirror registry that you created.
 `ocp-release`.
 <7> For `server_architecture`, specify the architecture of the server, such as `x86_64`.
 
+. Review the images and configuration manifests to mirror:
++
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --dry-run
+----
+
+. Mirror the version images to the internal container registry:
+** If your mirror host does not have internet access, take the following actions:
+... Connect the removable media to a system that is connected to the internet.
+... Mirror the images to a directory on the removable media:
++
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+----
+... Take the media to the restricted network environment and upload the images to the local container registry.
++
+----
+$ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror 'file://openshift/release:${OCP_RELEASE}*' ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}
+----
++
+[IMPORTANT]
```

Comment:
About how to capture imageContentSources in mirror-to-disk scenario, I already got a resolution from https://bugzilla.redhat.com/show_bug.cgi?id=1806779#c9, And QE's automation script is following such way to get it working well.

Reply:
@jianlinliu, can you be more specific? Trevor said that the enhancement to output the imageContentSources was rejected.

Reply:
Take an example here.
Firstly run mirror-to-registry command with '--run-dry' option to capture the imageContentSources output.
$ oc adm -a ${LOCAL_SECRET_JSON} release mirror \
     --from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} \
     --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} \
     --to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE} --run-dry
Then run mirror-to-disk command to upload the real contents to registry, but have to be the same target repository like the above command.
$ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror 'file://openshift/release:${OCP_RELEASE}*' ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}